tools/deploy.sh

```diff
@@ -4,10 +4,11 @@
 set -e
 
 # Remove old build
-rm -rf dist/*
-rm -rf build/*
+rm -rf ../dist/*
+rm -rf ../build/*
 
 # Build
+cd ..
```

Comment:
I'm sorry that I haven't clarified how to use the scripts in the `tools` directory.

When I wrote the deploy script, I didn't intend for it to be run in the `tools` directory. The original paths work if you run `./tools/deploy.sh` in the root directory for this repo. This applies to all other scripts in `tools` as well